src/index.html

```diff
@@ -20,8 +21,465 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
+  <body class="page">
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="/src/images/logo.png"
+        class="header__logo-link"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item header__nav-item--apple">
+            <a
+              href="#"
+              class="header__nav-link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="header__nav-item header__nav-item--samsung">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="header__nav-item header__nav-item--smartphones">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="header__nav-item header__nav-item--tablets">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="header__nav-item header__nav-item--photo">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="header__nav-item header__nav-item--video">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
 
-  <body>
-    <h1>Catalog</h1>
+    <!-- #region main -->
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <section class="card__top">
+          <img
+            class="card__top-image"
+            src="../src/images/imac.jpeg"
```

Comment:
The `src` attribute for the card image should be checked to ensure it points to the correct file relative to the HTML file location. Verify the path is correct.